quickjs.c

```diff
@@ -26267,6 +26267,29 @@ static char *js_default_module_normalize_name(JSContext *ctx,
     int len;
 
     if (name[0] != '.') {
+        /* Backwards compatibility for stdlib rename. */
```

Comment:
@bnoordhuis LMK what you think about this. I'm not super happy with the placement, since it's in the core library, but that's where the default normalizer lives...